content/hi/version_control.md

```diff
@@ -0,0 +1,17 @@
+---
+title: वर्शन कण्ट्रोल (Version Control)
+status: Completed
+category: तकनीकी
+---
+
+## यह क्या है
+
+सोर्स कण्ट्रोल (या वर्शन कण्ट्रोल) किसी डोकुमेंट में परिवर्तनों को ट्रैक करने और प्रबंधित करने का अभ्यास है। यह एक ऐसी सिस्टम है जो समय के साथ फ़ाइल या फ़ाइलों के सेट में परिवर्तनों को रिकॉर्ड करती है ताकि आप बाद में स्पेसिफिक वर्शनों को याद कर सकें।
```

Comment:
यह एक ऐसा

Reply:
Ok thanks for the suggestion @jayesh-srivastava & @ShivamTyagi12345 
I am changing 
and can you suggest all the changes at one time then it would be better for me

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix